kubeflow/optimizer/api/optimizer_client.py

```diff
@@ -119,6 +121,47 @@ def get_job(self, name: str) -> OptimizationJob:
 
         return self.backend.get_job(name=name)
 
+    def get_job_logs(
+        self,
+        name: str,
+        trial: Optional[str] = None,
```

Comment:
Can we use `trial_name` here ?
```suggestion
        trial_name: Optional[str] = None,
```